sdks/java/harness/src/main/java/org/apache/beam/fn/harness/FnApiDoFnRunner.java

```diff
@@ -784,9 +812,8 @@ private ByteString encodeProgress(double value) throws IOException {
       default:
         // no-op
     }
-  }
 
-  private void startBundle() {
+    // TODO: Support caching state data across bundle boundaries.
```

Comment:
Could you please add a JIRA here, which elaborate this support with more details?

Reply:
Done